src/middlewared/middlewared/plugins/apps/ix_apps/portals.py

```diff
@@ -7,6 +7,12 @@ def normalized_port_value(scheme: str, port: int) -> str:
     return '' if ((scheme == 'http' and port == 80) or (scheme == 'https' and port == 443)) else f':{port}'
 
 
+def normalized_host_value(host: str) -> str:
+    if ':' in host:
```

Comment:
Why would we not use the `ipaddress` module here and actually check `if obj.version == 6`?